src/chart/sankey/sankeyLayout.js

```diff
@@ -82,13 +84,17 @@ function layoutSankey(nodes, edges, nodeWidth, nodeGap, width, height, iteration
  * Compute the value of each node by summing the associated edge's value
  *
  * @param {module:echarts/data/Graph~Node} nodes  node of sankey view
+ * @param {string} mode  mode of sankey view
  */
-function computeNodeValues(nodes) {
+function computeNodeValues(nodes, mode) {
     zrUtil.each(nodes, function (node) {
         var value1 = sum(node.outEdges, getEdgeValue);
         var value2 = sum(node.inEdges, getEdgeValue);
         var nodeRawValue = node.getValue() || 0;
         var value = Math.max(value1, value2, nodeRawValue);
+        if (mode === 'uneven') {
+          value = nodeRawValue || value;
+        }
         node.setLayout({value: value}, true);
     });
```

Comment:
The judging logic of `mode === 'uneven'` is suggested to move to the place before `zrUtil.each`.
```suggestion
    var isUnevenMode = mode === 'uneven';
    zrUtil.each(nodes, function (node) {
        var value1 = sum(node.outEdges, getEdgeValue);
        var value2 = sum(node.inEdges, getEdgeValue);
        var nodeRawValue = node.getValue() || 0;
        var value = Math.max(value1, value2, nodeRawValue);
        if (isUnevenMode) {
          value = nodeRawValue || value;
        }
        node.setLayout({value: value}, true);
    });
```